spec/models/recipe_spec.rb

```diff
@@ -12,4 +12,22 @@
     it {should have_many :recipe_ingredients}
     it {should have_many(:ingredients).through(:recipe_ingredients)}
   end
+
+  describe "instance methods" do
+    before(:each) do
+      @recipe1 = Recipe.create!(name: "Spaghetti", complexity: 2, genre: "Italian")
+      @ingredient1 = Ingredient.create!(name: "Ground Beef", cost: 2)
+      @ingredient2 = Ingredient.create!(name: "Salt", cost: 4)
+      @ingredient3 = Ingredient.create!(name: "Pasta", cost: 5)
+      @ingredient4 = Ingredient.create!(name: "Tomato", cost: 1)
+
+      RecipeIngredient.create!(recipe_id: @recipe1.id, ingredient_id: @ingredient1.id)
+      RecipeIngredient.create!(recipe_id: @recipe1.id, ingredient_id: @ingredient3.id)
+      RecipeIngredient.create!(recipe_id: @recipe1.id, ingredient_id: @ingredient4.id)
+    end
+
+    it "#total_cost" do
+      expect(@recipe1.total_cost).to eq(8)
```

Comment:
For model tests I think it's good to test against multiple objects - this is fine, but it would be better to have a 2nd recipe with no ingredients, and make sure that returns `0`.